src/test/java/step4/model/CarTest.java

```diff
@@ -0,0 +1,73 @@
+package step4.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class CarTest {
+    Car car = new Car("temp");
+
+    @ParameterizedTest
+    @ValueSource(ints = {4, 6, 8})
+    @DisplayName("이동하는 경우")
+    void move_ok(int randomValue) {
+        car.move(randomValue);
+        assertThat(car.getPosition()).isEqualTo(1);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {-1, 3, 2})
+    @DisplayName("정지하는 경우")
+    void move_not_ok(int randomValue) {
+        car.move(randomValue);
+        assertThat(car.getPosition()).isEqualTo(0);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {4, 5, 6})
+    void isMove_true_test(int input) {
+        assertThat(car.isMove(input)).isEqualTo(true);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 2, 3})
+    void isMove_false_test(int input) {
+        assertThat(car.isMove(input)).isFalse();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"", " ", "    "})
+    void Name_공백인경우(String names) {
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            new Car(names);
+        });
```

Comment:
에러 메세지를 확인해 볼 수도 있을 거 같아요! 😄 
```suggestion
        }).withMessageContaining("****** Name length error!!");
```